mindsdb_native/libs/data_sources/file_ds.py

```diff
@@ -186,16 +188,22 @@ def _getDataIo(self, file):
             data.seek(0)
             first_few_lines = []
             i = 0
+
+            # need to have some templates to deduce a dialect
+            # but it is not a good idea to deduce dialect by header row
+            # data[0]
             for line in data:
                 if line in ['\r\n','\n']:
                     continue
                 first_few_lines.append(line)
                 i += 1
```

Comment:
Again, was here before, not introduced by this pr, but much cleaener to do `for i, line in enumerate(data)` than to have a manual incrementing `i`.

Reply:
totally agree with this note. Done